src/Controller/DepartementController.php

```diff
@@ -32,21 +35,15 @@ public function index(): Response
         ]);
     }
 
-    public function show(): Response
+    public function show(array $parameters): Response
```

Comment:
Je pense qu'ici il faudrait utiliser un objet de type Request (HttpFundation) et faire $request->get('id) pour récupérer l'id

Reply:
Sinon tu peux directement faire 

```suggestion
    public function show(?int $id): Response
```

Normalement le router va faire le mapping tout seul

Reply:
ça marche, dis moi quelle solution tu préfères ?

Reply:
Je te laisse choisir celle que tu préfères :)

Reply:
Je vais tout refaire proprement avec HttpFundation ! J'essais de finir ça pour ce week ! Compliquées les vacances scolaires :D